src/main/daml/Daml/Finance/Interface/Claims/Types.daml

```diff
@@ -30,3 +30,11 @@ data Pending = Pending
     instrument : Deliverable
     amount : Decimal
   deriving (Eq, Show)
+
+-- | Type with data required to describe an Election
+data ElectionData = ElectionData
```

Comment:
I think it makes sense to use the `Event` type that is defined in https://github.com/digital-asset/daml-finance/blob/main/src/main/daml/Daml/Finance/Claims/Util/Lifecycle.daml#L28

It probably requires moving the type in here and using a different name as Event is already used, but it allows us to write more generic code